rel/config/release.exs

```diff
@@ -0,0 +1,87 @@
+use Mix.Config
+
+defmodule Environment do
+  @moduledoc """
+  This modules provides various helpers to handle environment metadata
+  """
+
+  def get(key), do: System.get_env(key)
+
+  def get_boolean(key) do
+    case get(key) do
+      "true" -> true
+      "1" -> true
+      _ -> false
+    end
+  end
+
+  def get_integer(key, default \\ 0) do
+    case get(key) do
+      value when is_bitstring(value) -> String.to_integer(value)
+      _ -> default
+    end
+  end
+
+  def exists?(key) do
+    case get(key) do
+      "" -> false
+      nil -> false
+      _ -> true
+    end
+  end
+
+  def version do
+    case Application.spec(:elixir_boilerplatecase, :vsn) do
+      nil -> Mix.Project.config()[:version]
+      version -> to_string(version)
+    end
+  end
+end
+
+# Configure application
+force_ssl = Environment.get_boolean("FORCE_SSL")
+scheme = if force_ssl, do: "https", else: "http"
+host = Environment.get("CANONICAL_HOST")
+port = Environment.get("PORT")
+
+config :elixir_boilerplate,
+  canonical_host: host,
+  force_ssl: force_ssl,
+  version: Environment.version()
+
+# Configure Repo with Postgres
+config :elixir_boilerplate, ElixirBoilerplate.Repo,
+  pool_size: Environment.get_integer("DATABASE_POOL_SIZE"),
+  ssl: Environment.get_boolean("DATABASE_SSL"),
+  url: Environment.get("DATABASE_URL")
+
+# Configure endpoint
+config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
+  debug_errors: Environment.get_boolean("DEBUG_ERRORS"),
+  http: [port: port],
+  secret_key_base: Environment.get("SECRET_KEY_BASE"),
+  session_key: Environment.get("SESSION_KEY"),
+  signing_salt: Environment.get("SIGNING_SALT"),
+  static_url: [
+    scheme: Environment.get("STATIC_URL_SCHEME"),
+    host: Environment.get("STATIC_URL_HOST"),
+    port: Environment.get("STATIC_URL_PORT")
+  ],
+  url: [scheme: scheme, host: host, port: port]
+
+# Configure Basic Auth
+if Environment.exists?("BASIC_AUTH_USERNAME") do
+  config :elixir_boilerplate,
+    basic_auth: [
+      username: Environment.get("BASIC_AUTH_USERNAME"),
+      password: Environment.get("BASIC_AUTH_PASSWORD")
+    ]
+end
+
+# Configure Sentry
+config :sentry,
+  dsn: Environment.get("SENTRY_DSN"),
+  environment_name: Environment.get("SENTRY_ENVIRONMENT_NAME"),
+  included_environments: [:prod],
+  root_source_code_path: File.cwd!(),
+  release: Environment.version()
```

Comment:
We could split it in both files

### `config/config.exs`

```elixir
# Configure Sentry
config :sentry,
  included_environments: [:prod],
  root_source_code_path: File.cwd!(),
  release: Environment.version()
```

### `rel/config/runtime.exs`

```elixir
# Configure Sentry
config :sentry,
  dsn: Environment.get("SENTRY_DSN"),
  environment_name: Environment.get("SENTRY_ENVIRONMENT_NAME")
```